config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml

```diff
@@ -4322,11 +4327,12 @@ presubmits:
           limits:
             cpu: 4
             memory: 6Gi
-  - name: pull-kubernetes-node-e2e-crio-cgrpv1-dra-kubetest2
+  - name: pull-kubernetes-node-e2e-crio-cgrpv1-dra-kubetest2 # experimental alternative to pull-kubernetes-node-e2e-crio-cgrpv1-dra
```

Comment:
@elieser1101: what is the status of these kubetest2 variants?

Reply:
As far as I can see they don't differ much from the correspondent kubernetes_e2e.py variants. I'd propose to drop kubernetes_e2e.py jobs.

Reply:
At the moment the kubetest2 variants are not working because of the `--label-filter` not suported. I started to look at the fix on kubetests2 https://github.com/kubernetes-sigs/kubetest2/issues/285

Reply:
@elieser1101 Both kubetest2 jobs are "green" in the test grid: 
- https://testgrid.k8s.io/sig-node-cri-o#pr-node-kubelet-crio-cgrpv1-dra-kubetest2
- https://testgrid.k8s.io/sig-node-cri-o#pr-node-kubelet-crio-cgrpv2-dra-kubetest2

Do you mean they're not working correctly / as expected?

Reply:
yeah both are green but its because test are not actually runring, we can see that from the [build.log](https://storage.googleapis.com/kubernetes-ci-logs/pr-logs/pull/127709/pull-kubernetes-node-e2e-crio-cgrpv1-dra-kubetest2/1867608475635290112/build-log.txt)

```
Ran 0 of 718 Specs in 0.023 seconds
SUCCESS! -- 0 Passed | 0 Failed | 0 Pending | 718 Skipped
PASS
```